_maps/_basemap.dm

```diff
@@ -0,0 +1,19 @@
+//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```

Comment:
This file shouldn't exist. I meant to delete this. I was trying to figure things out.